codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -220,11 +220,16 @@ pub mod evm {
         TransactionIndex,
         #[subenum(RpcTransactionField)]
         Hash,
+        #[subenum(RpcTransactionField)]
         From,
+        #[subenum(RpcTransactionField)]
         To,
         Gas,
+        #[subenum(RpcTransactionField)]
         GasPrice,
+        #[subenum(RpcTransactionField)]
         MaxPriorityFeePerGas,
+        #[subenum(RpcTransactionField)]
         MaxFeePerGas,
         CumulativeGasUsed,
         EffectiveGasPrice,
```

Comment:
We can allow more fields after moving to our own rpc client.

Reply:
Why can't we do all fields? Are the others only available in a different rpc method or something?

Reply:
Ethers simply doesn't expose them, or exposes them already in decoded way, which is difficult to handle with current schema set up.